style/listview.less

```diff
@@ -0,0 +1,59 @@
+.results {
+	width: 100%;
+	background-color: white;
+	text-align: left;
+	overflow: scroll;
+	position: absolute;
+	top: 66px;
+	bottom: 30px;
+	text-align: left;
+}
+
+.result {
+	width: 100%;
+	height: 49px;
+	margin: 0;
+	padding: 6px 10px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.result .info {
+	height: 100%;
+	overflow: hidden;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.result .info .title {
+	font-size: 15px;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+
+}
+
+.result .info .description {
+	font-size: 12px;
+	color: @colorLight;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+}
+
+.result .img {
+	height: 100%;
+	display: inline-block;
+	text-align: right;
+}
+
+.result .img img {
+	width: 38px;
+	height: 38px;
+	border-radius: 4px;
+}
+
+.result[nav-selected="true"] {
+	background-color: #eaf3ff;
+}
```

Comment:
Missing a line return here. 

I wish linting could catch that.

Reply:
not sure how many linting tools for less, maybe can try this https://github.com/lesshint/lesshint

out of scope now, can try doing this in other pr.